browser/extensions/brave_extension_functional_test.cc

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
@emerick 
This file was moved to work across multiple files. It was simply moved from:
` browser/extensions/brave_extension_provider_browsertest.cc`, it was split into .h and .cc though and included in brave/test/BUILD.gn